SecurityException: Not allowed to access thread group via Gremlin

Error when executing gremlin `graph.schema().propertyKey('name').asText().ifNotExist().create()`:

```java
2022-03-07 17:05:56 [gremlin-server-exec-3] [WARN] c.b.h.s.HugeSecurityManager - SecurityException: Not allowed to access thread group via Gremlin
2022-03-07 17:05:56 [gremlin-server-exec-3] [ERROR] c.b.h.b.Transaction - Failed to commit changes:
java.lang.SecurityException: Not allowed to access thread group via Gremlin
        at com.baidu.hugegraph.security.HugeSecurityManager.newSecurityException(HugeSecurityManager.java:412) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.security.HugeSecurityManager.checkAccess(HugeSecurityManager.java:183) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at java.lang.ThreadGroup.checkAccess(ThreadGroup.java:315) ~[?:1.8.0_111]
        at java.lang.Thread.init(Thread.java:391) ~[?:1.8.0_111]
        at java.lang.Thread.init(Thread.java:349) ~[?:1.8.0_111]
        at java.lang.Thread.<init>(Thread.java:675) ~[?:1.8.0_111]
        at java.util.concurrent.Executors$DefaultThreadFactory.newThread(Executors.java:613) ~[?:1.8.0_111]
        at org.tikv.shade.com.google.common.util.concurrent.ThreadFactoryBuilder$1.newThread(ThreadFactoryBuilder.java:163) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.<init>(ThreadPoolExecutor.java:612) ~[?:1.8.0_111]
        at java.util.concurrent.ThreadPoolExecutor.addWorker(ThreadPoolExecutor.java:925) ~[?:1.8.0_111]
        at java.util.concurrent.ThreadPoolExecutor.execute(ThreadPoolExecutor.java:1357) ~[?:1.8.0_111]
        at java.util.concurrent.ExecutorCompletionService.submit(ExecutorCompletionService.java:181) ~[?:1.8.0_111]
        at org.tikv.raw.RawKVClient.doSendBatchPut(RawKVClient.java:615) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at org.tikv.raw.RawKVClient.batchPut(RawKVClient.java:223) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at org.tikv.raw.RawKVClient.batchPutAtomic(RawKVClient.java:216) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at org.tikv.raw.RawKVClient.batchPutAtomic(RawKVClient.java:206) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at com.baidu.hugegraph.backend.store.tikv.TikvStdSessions$StdSession.commit(TikvStdSessions.java:249) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at com.baidu.hugegraph.backend.store.tikv.TikvStdSessions$StdSession.commit(TikvStdSessions.java:190) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at com.baidu.hugegraph.backend.store.tikv.TikvStore.commitTx(TikvStore.java:258) ~[hugegraph-tikv-0.12.0-jar-with-dependencies.jar:?]
        at com.baidu.hugegraph.backend.tx.AbstractTransaction.commitMutation2Backend(AbstractTransaction.java:322) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.backend.tx.IndexableTransaction.commit2Backend(IndexableTransaction.java:57) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.backend.tx.AbstractTransaction.commit(AbstractTransaction.java:238) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.backend.tx.AbstractTransaction.commitOrRollback(AbstractTransaction.java:377) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.backend.tx.AbstractTransaction.afterWrite(AbstractTransaction.java:344) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.backend.tx.SchemaTransaction.addSchema(SchemaTransaction.java:357) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.backend.cache.CachedSchemaTransaction.addSchema(CachedSchemaTransaction.java:214) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.backend.tx.SchemaTransaction.addPropertyKey(SchemaTransaction.java:119) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.StandardHugeGraph.addPropertyKey(StandardHugeGraph.java:694) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.schema.builder.PropertyKeyBuilder.lambda$createWithTask$0(PropertyKeyBuilder.java:152) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.schema.builder.AbstractBuilder.lockCheckAndCreateSchema(AbstractBuilder.java:76) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.schema.builder.PropertyKeyBuilder.createWithTask(PropertyKeyBuilder.java:135) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.schema.builder.PropertyKeyBuilder.create(PropertyKeyBuilder.java:161) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.schema.builder.PropertyKeyBuilder.create(PropertyKeyBuilder.java:46) ~[hugegraph-core-0.12.0.jar:0.12.0.0]
        at com.baidu.hugegraph.schema.builder.SchemaBuilder$create$0.call(Unknown Source) ~[?:?]
        at Script3.run(Script3.groovy:1) ~[?:?]
        at org.apache.tinkerpop.gremlin.groovy.jsr223.GremlinGroovyScriptEngine.eval(GremlinGroovyScriptEngine.java:674) ~[gremlin-groovy-3.4.3.jar:3.4.3]
        at org.apache.tinkerpop.gremlin.groovy.jsr223.GremlinGroovyScriptEngine.eval(GremlinGroovyScriptEngine.java:376) ~[gremlin-groovy-3.4.3.jar:3.4.3]
        at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:233) ~[?:1.8.0_111]
        at org.apache.tinkerpop.gremlin.groovy.engine.GremlinExecutor.lambda$eval$0(GremlinExecutor.java:266) ~[gremlin-groovy-3.4.3.jar:3.4.3]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_111]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_111]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_111]
```

--- a/src/main/java/com/baidu/hugegraph/tikv/TikvPlugin.java
+++ b/src/main/java/com/baidu/hugegraph/tikv/TikvPlugin.java
@@ -8,6 +8,9 @@ import com.baidu.hugegraph.backend.serializer.BinarySerializer;
 import com.baidu.hugegraph.backend.store.tikv.TikvOptions;
 import com.baidu.hugegraph.backend.store.tikv.TikvStoreProvider;
 import com.baidu.hugegraph.plugin.HugeGraphPlugin;
+import com.baidu.hugegraph.security.HugeSecurityManager;
+
+import org.tikv.raw.RawKVClient;
 
 public class TikvPlugin implements HugeGraphPlugin {
 
@@ -24,6 +27,7 @@ public class TikvPlugin implements HugeGraphPlugin {
         HugeGraphPlugin.registerOptions(TikvStoreProvider.TYPE, classPath);
         classPath = BinarySerializer.class.getName();
         HugeGraphPlugin.registerSerializer(TikvStoreProvider.TYPE, classPath);
+        HugeSecurityManager.ignoreCheckedClass(RawKVClient.class.getName());
     }
 
     @Override

